Jenkinsfile

```diff
@@ -4,7 +4,7 @@ pipeline {
    parameters {
          booleanParam(defaultValue: true, description: 'Execute pipeline?', name: 'shouldBuild')
          booleanParam(defaultValue: false, description: 'Run Only Smoke Test', name: 'smokeOnly')
-
+         booleanParam(defaultValue: false, description: 'Skip Integration Tests?', name: 'skipIntegrationTests')
```

Comment:
this new `skipIntegrationTests` parameter will show up in the Jenkins UI when you manually trigger a build, right? Are there any other ways to set it?

Reply:
There aren't other ways to set it. Since we only want to use it in the case of master and release builds we don't have a PR in which to add keywords.